PiwikTracker.php

```diff
@@ -1070,11 +1107,17 @@ public function getUrlTrackAction($actionUrl, $actionType)
      *
      * Allowed only for Super User, must be used along with setTokenAuth()
      * @see setTokenAuth()
-     * @param string $dateTime Date with the format 'Y-m-d H:i:s', or a UNIX timestamp
+     * @param string|DateTimeInterface $dateTime Date with the format 'Y-m-d H:i:s', or a UNIX timestamp, or DateTimeInterface object
+     * @return $this
      */
     public function setForceVisitDateTime($dateTime)
     {
-        $this->forcedDatetime = $dateTime;
+        if ($dateTime instanceof DateTimeInterface) {
```

Comment:
You are right, DateTimeInterface comes with 5.5.0. i didn't think backward compability. We can use `DateTime` instead of interface. What do you think? Maybe we can remove whole DateTime object passing if you think its not necessary.